FRINXio/frinx-frontend
# FR-311 mpls device details

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FR-311` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  |


IMPORTANT:
test with https://github.com/FRINXio/frinx-inventory-server/pull/461

HOW TO TEST:

click on mpls nodes -> thera are 2 buttons (MPLS, LSP) in info panel -> by clicking it you shoud see data table in modal
